src/res/translation/translation_pl_PL.ts

```diff
@@ -1459,27 +1461,27 @@ We muted your channel and activated &apos;Mute Myself&apos;. Please solve the fe
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="285"/>
         <source>Input Boost</source>
-        <translation type="unfinished"></translation>
+        <translation>Wzmocnienie Wejścia</translation>
     </message>
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="286"/>
         <source>This setting allows you to increase your input signal level by factors up to 10 (+20dB).If your sound is too quiet, first try to increase the level by getting closer to the microphone, adjusting your sound equipment or increasing levels in your operating system&apos;s input settings. Only if this fails, set a factor here. If your sound is too loud, sounds distorted and is clipping, this option will not help. Do not use it. The distortion will still be there. Instead, decrease your input level by getting farther away from your microphone, adjusting your sound equipment or by decreasing your operating system&apos;s input settings.</source>
-        <translation type="unfinished"></translation>
+        <translation>Ta opcja pozwala na stopniowe zwiększenie poziomu twojego sygnału wejściowego aż do 10 (+20dB). Kiedy twój dźwięk jest zbyt cichy, najpierw spróbuj zwiększyć głośność zbliżając się do mikrofonu, ustawiając lepiej sprzęt audio lub zwiększając poziomy w ustawieniach wejścia dźwięku w swoim systemie operacyjnym. Dopiero gdy wszystko powyższe zawiedzie, ustaw to wzmocnienie tutaj. Jednak gdy twój dźwięk jest zbyt głośny, zniekształcony lub urywany, ta opcja nie pomoże. Wtedy jej nie używaj. Zniekształcenia i tak pozostaną. Zamiast tego obniż poziom sygnału wejściowego odsuwając się od mikrofonu, zmieniając ustawienia twoich urządzeń dźwiękowych lub ściszając poziomy wejścia w swoim systemie.</translation>
     </message>
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="300"/>
         <source>Input Boost combo box</source>
-        <translation type="unfinished"></translation>
+        <translation>Kontrolka Wzmocnienia Wejścia</translation>
     </message>
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="304"/>
         <source>Leave this blank unless you need to enter the address of a directory server other than the default.</source>
-        <translation>Należy pozostawić to puste pole, chyba że konieczne jest wprowadzenie adresu centralnego serwera innego niż domyślny.</translation>
+        <translation>Należy pozostawić to puste pole, chyba że konieczne jest wprowadzenie adresu serwera innego niż domyślny.</translation>
     </message>
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="309"/>
         <source>Directory server address combo box</source>
-        <translation>Lista wyboru serwera  centralnego</translation>
+        <translation>Lista wyboru serwera</translation>
```

Comment:
As far as I understand, this lacks the translation for "Directory", is my understanding correct and is this by intention?

Reply:
It would not harm much if it stay like this, but let's have it tiptop :-)
Patch is coming...